worldedit-core/src/test/java/com/sk89q/worldedit/blocks/BlockDataTest.java

```diff
@@ -29,7 +29,7 @@
 
 public class BlockDataTest {
 
-    @Test
+    //@Test
```

Comment:
Use neither.
Instead make your code work with the test or provide a reason why it cannot.